### PR TITLE
Gidk 27by7a heading stage

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Comment by shalinee-walkover** [comment]
https://app.clickup.com/t/27by7a
Developer -  @ravinderwalkover @shalinee-walkover 

Description - Heading issue resolved, Heading should be visible on small screen

File: apps/web-giddh/src/app/settings/settings.component.ts [hunk 1]
```diff
@@ -39,6 +39,8 @@ export class SettingsComponent implements OnInit, OnDestroy {
     @ViewChild('permissionComp', { static: false }) public permissionComp: SettingPermissionComponent;
     @ViewChild('tagComp', { static: false }) public tagComp: SettingsTagsComponent;
 
+
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/settings.component.ts:42]: remove space

File: apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts [hunk 2]
```diff
@@ -7,6 +7,10 @@ import { CompanyResponse, StateDetailsRequest } from '../models/api-models/Compa
 import { CompanyActions } from '../actions/company.actions';
 import { ReplaySubject } from 'rxjs';
 import { TabsetComponent } from 'ngx-bootstrap/tabs';
+<<<<<<< HEAD
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts:10]: resolve conflict

File: apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts [hunk 3]
```diff
@@ -41,15 +47,50 @@ export class TbPlBsComponent implements OnInit, OnDestroy {
     /* This will hold common JSON data */
     public commonLocaleData: any = {};
 
+<<<<<<< HEAD
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts:50]: resolve conflict

File: apps/web-giddh/src/app/userDetails/userDetails.component.html [hunk 4]
```diff
@@ -5,7 +5,11 @@
     </div>
 
     <div class="setting-page user-detail-page relative" id="setting-tab" *ngIf="!isCreateAndSwitchCompanyInProcess">
+<<<<<<< HEAD
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/userDetails/userDetails.component.html:8]: resolve conflict

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**